src/patterns/components/promo/has-icon.hbs

```diff
@@ -1,9 +1,18 @@
 ---
 name: Promo Image With Icon
 description:
-  Promos can be used with an optional clickable illustration or icon.
+  Promo with optional clickable icon.
 restrictions:
- - Illustration/icon must have a descriptive alt tag if it's an image or a descriptive title tag if the format is SVG.
+ - Icon must have a descriptive title tag if the format is SVG.
 order: 2
 ---
-{{#embed "patterns.components.promo.base" promoIcon=true promoBtnLink=true promoLink=false promoImg=false}}{{/embed}}
+<div class="sprk-c-Promo sprk-o-Stack sprk-o-Stack--split@s">
+  <a href="#" class="sprk-o-Stack__item sprk-o-Stack__item--equal@s sprk-c-Promo__icon">
+    <svg height="150" width="167" viewBox="0 0 220.63 197.62">
+      <use xlink:href="#sparkler" />
```

Comment:
The sparkler icon is no longer available (i removed it). I wonder if your branch needs to be rebased?

Reply:
I think this (and the color variable, and the angular component) are all because I was based off of fall and not staging.

Reply:
Sorry, I think we dropped the ball on communicating this because we stopped using that branch when you were gone.

Reply:
Yeh, ugh, sorry @bobwise, that was our fault.